rules/static_framework_generator.bzl

```diff
@@ -0,0 +1,214 @@
+"""This file contains rules to build framework binaries from your podfile or cartfile"""
+
+load("@build_bazel_rules_apple//apple:providers.bzl", "AppleResourceInfo")
+
+def static_framework_generator(**kwargs):
```

Comment:
I dont think naming the rule `_generator` makes much sense to me. it's implicit that rules generate outputs

Reply:
I can change the name. I prefer to avoid naming it `ios_static_framework` so the name does not clash with the existing [`ios_static_framework`](https://github.com/bazelbuild/rules_apple/blob/master/doc/rules-ios.md#ios_static_framework).

Any proposals?